prefabs/test1.json

```diff
@@ -0,0 +1,7 @@
+{
+    "tiles":
+    [
+        [0, 0],
+        [1, 1]
+    ]
+}
```

Comment:
Just an Idea
To prevent adding to the project a lot of test jsons files (I will also be using that a lot), maybe we can have a funcrion to parse a raw string, so for testing we will only need to add:
```
string input = R"({
    "tiles":
    [
        [0, 0],
        [1, 1]
    ]
} )";
```
inside theprefabTest code.
The "createPrefabFromJson" can just dump the file into a string and call
the raw string function